various_fixes

Tatsukio opened the pull request.

Kr1s1m reviewed (commented):
What do these modifications change/fix? "various fixes" - write them down in the long description form - use github desktop to make this process easy

Tatsukio commented:
Info.usl
Fixed empty texture (Error in blend spec: ' layer mirage_empty_card') and wrong GetBlendBitmap string

attribs_def.txt
Fixed wrong variables data types and names

FightingObj.usl
Fixed variables names
